@app/db/scripts/test-seed.js

```diff
@@ -18,7 +18,10 @@ async function main() {
   }
 }
 
-main().catch((e) => {
-  console.error(e);
-  process.exit(1);
-});
+const inTests = process.env.IN_TESTS;
+if (inTests) {
```

Comment:
Please move this block either to the top of the file, or to the top of the `main` function body.